src/main/java/org/radarbase/authorizer/service/managementportal/CachedManagementPortalClient.java

```diff
@@ -146,7 +169,7 @@ private Subject querySubject(String subjectId) throws IOException, TokenExceptio
 
   private Project queryProject(String projectId) throws IOException, TokenException {
     Project project = queryEntity(
-        properties.getBaseUrl() + "/api" + properties.getProjectsPath() + "/" + projectId,
+        properties.getBaseUrl() + properties.getProjectsPath() + "/" + projectId,
```

Comment:
why do we need to configure project paths etc ? can't we just configure managementportal baseUrl and you add the string here?